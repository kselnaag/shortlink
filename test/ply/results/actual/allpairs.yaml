allpairs:  # 22.10.2023, 19:19:03:016
  request:
    url: http://localhost:8080/check/allpairs
    method: GET
    headers: {}
  response:  # 44 ms
    status:
      code: 200
      message: OK
    headers:
      cache-control: no-cache
      connection: close
      content-length: '115'
      content-type: application/json; charset=utf-8
    body: |-
      {
        "Body": "5clp60: http://lib.ru; 8b4s29: http://lib.ru/PROZA/; dhiu79: http://google.ru",
        "IsResp": true,
        "Mode": "200"
      }
