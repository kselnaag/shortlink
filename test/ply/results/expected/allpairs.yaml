allpairs:
  request:
    url: ${baseUrl}/check/allpairs
    method: GET
    headers: {}
  response:
    status:
      code: 200
      message: OK
    headers:
      cache-control: no-cache
      connection: close
      content-length: '115'
      content-type: application/json; charset=utf-8
    body: |-
      {
        "Body": "5clp60: http://lib.ru; 8b4s29: http://lib.ru/PROZA/; dhiu79: http://google.ru",
        "IsResp": true,
        "Mode": "200"
      }
